docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -378,7 +374,7 @@ storage account so other people on your team can access it via `make secrets`
 # Environment file
 
 1. Choose the resource group prefix.  The resource group location will be
-   appended to the prefix to make the resource group name.
+   appended to the prefix to make the resource group name. Suggestion is to be unique just going with "v4" initially could run you into collisions with others depending on if there are existing resources.
```

Comment:
Thoughts on going with something like this?
```suggestion
   The resource group location will be appended to the prefix to make the resource group name. If a v4-prefixed environment exists in the subscription already, use a unique prefix.
```

Reply:
Excellent!